dev/io.openliberty.jakarta.mail.2.0/src/io/openliberty/jakarta/mail/internal/injection/MailSessionDefinitionInjectionBinding.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2015,2017,2020
+ * Copyright (c) 2015, 2020
```

Comment:
Sorry but it really does have to include `IBM Corporation and others.` ... i.e. who's copyright is it?

Reply:
Hi Tom, The comment was originally from the 1.5 version of java mail. I've only tweaked it for 1.6 an 2.0 so I've assumed that "others" was appropriate but you may have a good point that in this case IBM Corporation may be the correct  answer.

Reply:
The vast majority of our Open Liberty copyright headers say "IBM Corporation and others.". Just use that.